### DISCUSSION TITLE
Push My dev to remote dev already

### DISCUSSION THREAD

**Discussion by ZIHAOJIA** [MichaelKelly9/Tafe24S1T104_Ext#8] [General]
Hi, just let you know ,I have pushed my dev to remote dev already

**Comment by MichaelKelly9** [comment]
Thanks @ZIHAOJIA ,

I will now pull the remote Dev branch and do the merge of my feature. I will advise once it is complete and pushed back into Github

**Comment by MichaelKelly9** [comment]
Hi @ZIHAOJIA 
I have merged my feature and pushed the latest Dev into Github Repo. 

Please have a look and confirm it is all functioning as expected or if there is any styling that needs to be changed.

I also updated the main menu buttons to function correctly but have just realized that coding was supposed to be one in the release branch (Part D). But i don't think it will matter if we did it with this commit anyhow.

**Reply by ZIHAOJIA** [reply]
I pull the dev and it works well!
